src/factories/requestFactory.ts

```diff
@@ -28,7 +28,7 @@ export const assembleCreateNewAssetRequest = (
       addressLine2: values?.addressLine2 ?? "",
       addressLine3: values?.addressLine3 ?? "",
       addressLine4: values?.addressLine4 ?? "",
-      postCode: values.postcode,
+      postCode: values.postcode.toUpperCase(),
```

Comment:
Do you still need this if .uppercase() is used?

Reply:
Yeah, that uppercase() in the schema just takes the value, transforms it to upper case and compares it against the regex for validation purposes, but it does not change the actual value